bench/GEMMsBenchmark.cc

```diff
@@ -27,19 +27,23 @@
 using namespace std;
 using namespace fbgemm;
 
-void performance_test() {
+void performance_test(int argc, const char **argv) {
+  const int M = parseArgumentInt(argc, argv, "--M=", 0, 0);
+  const int N = parseArgumentInt(argc, argv, "--N=", 0, 0);
+  const int K = parseArgumentInt(argc, argv, "--K=", 0, 0);
   // clang-format off
-  static const vector<vector<int>> shapes = {
+  const vector<vector<int>> shapes = {
     // NOTE: clang-format wants to use a different formatting but the current
     // formatting should be easier to read.
     // m, n, k
-    {64, 800, 320},
-    {64, 768, 512},
-    {16, 256, 512},
-    {128, 128, 128},
-    {256, 512, 256},
-    {1024, 1024, 1024},
+    {M?M:64, N?N:800, K?K:320},
+    {M?M:64, N?N:768, K?K:512},
+    {M?M:16, N?N:256, K?K:512},
+    {M?M:128, N?N:128, K?K:128},
+    {M?M:256, N?N:512, K?K:256},
+    {M?M:1024, N?N:1024, K?K:1024},
```

Comment:
We should parse `argc` and `argv` before calling `performance_test`, and just pass `M`, `N`, `K` to `performance_test` instead

Reply:
ok

Reply:
done